test/message/async_error_sync_esm.out

```diff
@@ -5,4 +5,3 @@ Error: test
     at async three (*fixtures*async-error.js:20:3)
     at async four (*fixtures*async-error.js:24:3)
     at async main (*message*async_error_sync_esm.mjs:7:5)
-(node:*) [DEP0130] DeprecationWarning: Module.createRequireFromPath() is deprecated. Use Module.createRequire() instead.
```

Comment:
Why this patch cause this line removed ?

Reply:
Probably because usage of `createRequireFromPath()`, which causes the deprecation warning, is being removed from `test/common/index.mjs`.

Reply:
Yep, exactly. The tests used to use a deprecated method which seemed needlessly confusing. It’s a bit of a drive-by change and I’d be happy to pull it out if there’s demand. :)